lib/Dialect/SPIRV/Serialization/Serializer.cpp

```diff
@@ -618,6 +636,29 @@ Serializer::prepareBasicType(Location loc, Type type, uint32_t resultID,
     return success();
   }
 
+  if (auto structType = type.dyn_cast<spirv::StructType>()) {
+    for (uint32_t elementNum = 0; elementNum < structType.getNumElements();
```

Comment:
Same as above, don't recompute the end of the loop.

Reply:
Thanks.